### PR TITLE
Equalizes noble loadout

### PR DESCRIPTION
**Pull request by anything86**
## About The Pull Request
Equalizes noble loadout so one of them isn't getting over 100 mammon more in unique gear.
Also equalizes the assassin disguise's loadout, since the male version was just far more useful.

## Why It's Good For The Game
Something something gender equality. It's just silly women were getting a steel dagger worth 30 mammon while men got a golden rapier worth 140. Swords are cool, rapiers are doubly as cool.
Also you could literally just, sell the gold rapier and buy a crossbow, bolts and a dagger if you wanted, on the assassin's end.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

### CODE REVIEW

**Comment by BadAtThisGame302** [comment]
I think it was because men weren't supposed to have a bow, but somone added it to the men and forgot that's now how it works.

**Comment by cornercube** [comment]
> I think it was because men weren't supposed to have a bow, but somone added it to the men and forgot that's now how it works.

when I added the noble class, both of them only had daggers, since they were meant to be outright weak in combat in exchange for an INT bonus and more coin
though the intent of the noble archetype probably drifted over time

**Comment by NPC1314** [comment]
Just having fancy daggers feels enough tbh, I think theyre cooler than rapiers.
If they absolutly got to have both golden rapiers and lots of dosh have combat skills at 1 at most and no armor or something,

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/noble.dm [hunk 1]
```diff
@@ -42,13 +42,13 @@
 	id = /obj/item/clothing/ring/silver
 	if(H.gender == FEMALE)
 		H.change_stat("speed", 1)
-		H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
```
Comment (NPC1314) [inline code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/noble.dm:45]: should be 1

**Comment by NPC1314** [comment]
Nobles need a overhaul, 200 mammon + 140 mammon rapier for some rando pilgrim is a bit much, lets sort it another way.

**Pull request closed without merging**